### PR TITLE
add antiscroll@0.9 w/ git auto-update

### LINKED ISSUE #8434: [Request] Add antiscroll
**Library name:** antiscroll
**Git repository url:** https://github.com/Automattic/antiscroll
**npm package url(optional):** 
**License(s):**
**Official homepage:** http://automattic.github.io/antiscroll/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by nolsherry**
PR for #8434 
@pvnr0082t , @kennynaoh , please help me review this PR, thanks!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/Automattic/antiscroll/
- Official website (optional, not the repository): https://automattic.github.io/antiscroll/
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars: 1039
  - Count of watchers: 216
  - Count of forks: 171
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [X] This lib was not found on cdnjs repo
- [X] No already exist / duplicated issue and PR
- [X] The lib has notable popularity
  - [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [X] Has valid tags for each versions (for git auto-update)
- [X] Auto-update setup
- [X] Auto-update target/source is valid.
- [X] Auto-update filemap is correct.
# Git commit checklist
- [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [X] The parent of the commit(s) in the PR is not old than 3 days.
- [X] Pull request is sending from a non-master branch with meaningful name.
- [X] Separate unrelated changes into different commits.
- [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [X] Close corresponding issue in commit message
- [X] Mention related issue(s), people in commit message, comment.

close #8434, cc @nylen

### CODE REVIEW

File: ajax/libs/antiscroll/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "version": "0.9",
+  "description": "Antiscroll addresses this issue by providing a cross-browser implementation of the scrollbars popularized by OS X Lion that retains native properties.",
+  "author": {
+    "name": "James Nylen",
+    "url": "https://github.com/nylen"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/nylen/antiscroll"
```
Comment (pvnr0082t) [inline ajax/libs/antiscroll/package.json]: @nolsherry where did you get this url? This link shows `404 not found`.

Comment (nolsherry) [inline ajax/libs/antiscroll/package.json]: @pvnr0082t, sorry for making this mistake, I mistook the `url` in `repository` field, and I modified `url`, thanks for your help!

**Comment by nolsherry** [comment]
@pvnr0082t , @kennynaoh , I have modified the wrong `url` into the correct `url` in `package.json`,
Please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by nolsherry** [comment]
@pvnr0082t , no problem, I will contribute `package.json` to upstream after this PR is merged successfully.

**Comment by PeterDaveHello** [comment]
what about scrollbar.js?

**Comment by nolsherry** [comment]
@PeterDaveHello, I will confirm with author whether the file is necessary.

**Comment by nolsherry** [comment]
@PeterDaveHello the author does not response me for a while ([antiscroll/issues#97](https://github.com/Automattic/antiscroll/issues/97)), should we add `scrollbar.js` ?

**Comment by PeterDaveHello** [comment]
Maybe we can just add that file.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by nolsherry** [comment]
@PeterDaveHello sorry for the late reply, I have added `scrollbar.js`.
@kennynaoh I have modified `package.json`, and added  `scrollbar.js`,
please help me review this PR, thanks!

**Pull request merged**